config/config_params.go

```diff
@@ -510,6 +537,13 @@ func loadParams() {
 		case "iface-list":
 			param = &RegexpParam{Regexp: IfaceListRegexp,
 				Msg: "invalid Linux interface name"}
+		case "iface-list-regexp":
+			param = &RegexpPatternListParam{
+				ListRegexp: RegexpIfaceListRegexp,
+				ElemRegexp: RegexpIfaceElemRegexp,
+				Delimiter:  ",",
+				Msg:        "list contains invalid Linux interface name or regex pattern",
+			}
```

Comment:
Turned the parameter into a new category, since it doesn't rely on just a regexp value for validation.